src/osgPlugins/ac/ac3d.cpp

```diff
@@ -45,6 +45,7 @@ namespace ac3d {
 osg::Node*
 readFile(std::istream& stream, const osgDB::ReaderWriter::Options* options);
 
+static std::string ac3dSrcFilename;
```

Comment:
Use of static's introduce threading issues, the only way to avoid it would be to use a mutex around the reading and writing of the static variable.  Given this variable looks only there for some debug messages I believe i't really inappropriate change. 

My recommendation would be to ditch all the changes to the ac plugin.  The other change look acceptable.